spec/views/layouts/application.html.erb_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'layouts/application', type: :view do
+  let(:mock_patron) { instance_double Patron, id: 'idhere' }
+
+  before do
+    without_partial_double_verification do
+      allow(view).to receive(:patron).and_return(mock_patron)
+    end
+  end
+
+  context 'when site is in not in maintenance mode' do
+    after do
+      Settings.maintenance_mode = false
+    end
+
+    it 'makes a child checkbox with that name' do
```

Comment:
description needs to be updated here as well

Reply:
🤭